ams/server/src/main/java/com/netease/arctic/server/table/executor/BaseTableExecutor.java

```diff
@@ -50,7 +51,7 @@ protected BaseTableExecutor(TableManager tableManager, int poolSize) {
             poolSize,
             new ThreadFactoryBuilder()
                 .setDaemon(false)
-                .setNameFormat("ASYNC-" + getThreadName() + "-%d")
+                .setNameFormat("async-" + StringUtils.lowerCase(getThreadName()) + "-%d")
```

Comment:
Same here.

Reply:
Done, thanks for your useful suggestion.